spec.md

```diff
@@ -128,6 +128,55 @@ Any new types and keywords added to future PHP versions MUST be in lower case.
 Short form of type keywords MUST be used i.e. `bool` instead of `boolean`,
 `int` instead of `integer` etc.
 
+Compound types includes intersection, union, and mixed intersection and union type declarations. PHP requires
+that all compound types be structured as an ORed (unioned) series of ANDs (intersections), and that each set of
+intersections be encased with parentheses.
+
+The union symbol `|` and intersection symbol `&` MUST NOT have a leading or trailing space.  The parentheses MUST NOT
```

Comment:
While I personally agree that there should be no spaces around the "symbols", this appears to be inconsistent with `catch()` which has spaces around in the example:

https://github.com/php-fig/per-coding-style/blob/master/spec.md?plain=1#L967

Reply:
That should unified one way or another 👍